tools/bmh_test/create_bmh.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -x
+
+# -------------------------------------------------------------------------------------------
+# Description:    This script creates a virtual machine using virt-install,
+#                 adds the virtual machine to VBMC (Virtual BMC) for out-of-band management,
+#                 and applies the configuration to Kubernetes
+#
+# Usage:          make tilt-up -> press button in the right upper corner to create bmhs
+#                 /tools/bmh_test/create_bmh.sh
+# 
+# Prerequecites:  a network with ip address of 192.168.222.1 named bmo-test-network and
+#                 VBMC runing
+# -------------------------------------------------------------------------------------------
+
+set -xe
+
+# Set default values for VM name and VBMC port
+name="${1:-host-0}"
+vbmc_port="${2:-16230}"
+consumer="${3}"
+consumer_namespace="${4}"
+
+# Generate a random MAC address for the VM's network interface
+mac_address="$(printf '00:60:2F:%02X:%02X:%02X\n' $((RANDOM%256)) $((RANDOM%256)) $((RANDOM%256)))"
+
+# Create a virtual machine using virt-install
+virt-install \
+  --connect qemu:///system \
+  --name "${name}" \
+  --description "Virtualized BareMetalHost" \
+  --osinfo=ubuntu-lts-latest \
+  --ram=4096 \
+  --vcpus=2 \
+  --disk size=20 \
+  --graphics=none \
+  --console pty \
+  --serial pty \
+  --pxe \
+  --network network=bmo-test-network,mac="${mac_address}" \
+  --noautoconsole
+
+# Add the BareMetalHost VM to VBMC
+docker exec vbmc vbmc add "${name}" --port "${vbmc_port}" --libvirt-uri "qemu:///system"
+docker exec vbmc vbmc start "${name}"
+docker exec vbmc vbmc list
+
+# Create a YAML file to generate Kubernetes configuration for the VM
+# Apply the generated YAML file to the cluster
+if [[ "${consumer}" && "${consumer_namespace}" ]]; then
+  echo "Applying YAML for controlplane host..."
+  cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: $name-bmc-secret
```

Comment:
```suggestion
  name: ${name-bmc-secret}
```
For readability and consistency. This repeats a lot in this file, I'll just write it once here.